app/javascript/src/components/Dashboard/Notes/NoteList/NoteDropdown.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { Dropdown } from "neetoui";
+
+import DropdownMenu from "./DropdownMenu";
+
+const NoteDropdown = ({ handleEditClick, handleDeleteClick }) => {
+  const { Menu, MenuItem } = Dropdown;
+  return (
+    <Dropdown customTarget={DropdownMenu}>
+      <Menu>
+        <MenuItem.Button onClick={handleEditClick}>Edit</MenuItem.Button>
+        <MenuItem.Button onClick={handleDeleteClick}>Delete</MenuItem.Button>
```

Comment:
Pass style as danger for irreversible actions.

Reply:
Yes, Yedhin. Thanks for pointing this out.